packages/@aws-cdk/aws-apigateway/lib/integration.ts

```diff
@@ -71,7 +71,7 @@ export interface IntegrationOptions {
    * the key, and the template is the value (specified as a string), such as
    * the following snippet:
    *
-   *   { "application/json": "{\n  \"statusCode\": \"200\"\n}" }
+   *   { "application/json": "{\n  \"statusCode\": 200\n}" }
```

Comment:
Needs a codeblock fence around the example as well if you're improving docs anyway.